README.md

```diff
@@ -13,11 +13,11 @@ check out our [PythonAnywhere demo site](http://defectdojo.pythonanywhere.com/).
 
 You can log in as an administrator like so:
 
-![Admin](https://inventropy.us/dd/admin.png) with password **administrator**
+![Admin](https://raw.githubusercontent.com/OWASP/django-DefectDojo/master/doc/img/admin.png)
 
 You can also log in as a product owner / non-staff user:
 
-![Product owner](https://inventropy.us/dd/owner.png) with password **product_user**
+![Product owner](https://raw.githubusercontent.com/OWASP/django-DefectDojo/master/doc/img/owner.png)
```

Comment:
Hi @mattvaldes, there is a known issue with markdown images in GitHub so I would recommend we leave this as is. Same thing as before :)